upload-source-maps/action.yml

```
@@ -32,7 +44,7 @@ runs:
 
         if [[ $(yarn davinci-engine help) == *"sentry-upload-source-maps"* ]]
         then
-          yarn davinci-engine sentry-upload-source-maps
+          yarn davinci-engine sentry-upload-source-maps --url-prefix="$URL_PREFIX" --app-build-directory="$APP_BUILD_DIRECTORY" --release-name="$RELEASE_NAME"
```

what if `inputs.url-prefix` is not passed, whether it will work properly as expected?

---

Yes, it will work @dmaklygin as `urlPrefix` will be passed as empty string, and in this conditional it will be falsy:
```
  if (urlPrefix) {
    args.push('--url-prefix', urlPrefix)
  }
```

---

You could do it from bash the shell script directly, we have some similar example on how to do it on https://github.com/toptal/davinci-github-actions/blob/45abda9f3e03d1164d1cbc799d51470d4dbdd77a/build-publish-alpha-package/action.yml#L54

There's a special variable expansion, that only expands when some variable is present `${var+expansion}`, so it would be something like, also splitting lines for readability:

```suggestion
          yarn davinci-engine sentry-upload-source-maps \
            ${URL_PREFIX+"--url-prefix="$URL_PREFIX"} \
            ${APP_BUILD_DIRECTORY+"--app-build-directory=$APP_BUILD_DIRECTORY"} \
            ${RELEASE_NAME+"--release-name=$RELEASE_NAME"}
```